src/main/webapp/portal/main/partials/sidebar-left.html

```diff
@@ -10,6 +10,19 @@
     </div>
     <hr>
     <div ng-controller="PortalSidebarController as sidebarCtrl">
+    	<ul class="list-group sidebar-list">
+          <li class="list-group-item" ng-click="headerCtrl.navbarCollapsed = true;">
+            <a href="/" target="_self" ng-click="pushGAEvent('Sidebar', 'Click Link', 'Home');">
+              <span class="fa fa-home fa-fw"></span> Home
+            </a>
+          </li>
+          <li class="list-group-item" ng-click="headerCtrl.navbarCollapsed = true;">
+            <a href="{{MISC_URLS.logoutURL}}" target="_self" ng-click="pushGAEvent('Sidebar', 'Click Link', 'Log out');">
+              <span class="fa fa-sign-out fa-fw"></span> Log out
+            </a>
+          </li>
+       </ul>
+       <hr/>
         <ul class="list-group sidebar-list">
           <li class="list-group-item" ng-repeat="item in sidebar" ng-if="sidebarCtrl.canSee(item.displayFlag)" ng-click="headerCtrl.navbarCollapsed = true;">
```

Comment:
If this `ng-repeat` (or `ng-if`) were to filter out `sidebar` items titled `"Home"` and `"Log out"`, then the `frame` with this changeset could be upgraded to with less update by inheriting `frame`-based applications, that is, they would have less need to update their `sidebar.json`.  Might not be worth the complexity cost to achieve a less-change migration path.